src/main/java/com/egen/AppInitialize.java

```diff
@@ -9,7 +9,7 @@
 
 	@Override
 	protected Class<?>[] getRootConfigClasses() {
-		return new Class<?>[0];
+		return new Class<?>[]{AppConfig.class};
```

Comment:
missing JPAConfig.class. This is the entry point in the application so you specify all the config classes you want to execute here.